bundles/org.openhab.binding.insteon/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -459,11 +465,21 @@
 		<label>System Mode</label>
 	</channel-type>
 
+	<channel-type id="tamperSwitch">
+		<item-type>Contact</item-type>
+		<label>Tamper Switch</label>
+	</channel-type>
+
 	<channel-type id="temperature">
 		<item-type>Number:Temperature</item-type>
 		<label>Temperature</label>
 	</channel-type>
 
+	<channel-type id="temperatureLevel">
+		<item-type>Number</item-type>
+		<label>Temperature Level</label>
+	</channel-type>
```

Comment:
How is temperature level different from temperature?

Reply:
It's described in the README.md. it's used as part of the formula to calculate the actual temperature